core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -291,14 +291,21 @@ object ConsumerGroupCommand extends Logging {
     }
 
     private def printStates(states: Map[String, GroupState]): Unit = {
-      for ((groupId, state) <- states) {
-        if (shouldPrintMemberState(groupId, Some(state.state), Some(1))) {
+      val stateProps =  states.filter{case(groupId,state)=>shouldPrintMemberState(groupId, Some(state.state), Some(1))}
```

Comment:
I wonder if we shouldn't simply get rid of `shouldPrintMemberState` and print out all the groups in the table. `shouldPrintMemberState` seems to log some info based on the state of the group. I think that the state would be self explanatory in the table. Would this make sense?

Reply:
I wanted to avoid changing any existing behaviour with this PR. In light of this, could we print all the groups regardless of their state when the `--all-groups` flag is given?